addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -4206,6 +4206,67 @@ QUnit.test('auto-focus composer on opening thread', async function (assert) {
     );
 });
 
+QUnit.test('mark channel as seen if last message is visible when switching channels [REQUIRE FOCUS]', async function (assert) {
+    assert.expect(1);
+
+    // channel expected to be found in the sidebar, with the expected message_unread_counter
+    // and a random unique id that will be referenced in the test
```

Comment:
You can remove those comments, we knew it was a bad idea even when I added them, they are just too verbose and makes reading the test more complicated than just trying to figure out what the variables are.